PURL-TYPES.rst

```diff
@@ -258,15 +258,13 @@ golang
 
 - There is no default package repository: this is implied in the namespace
   using the ``go get`` command conventions.
-- The ``namespace`` and `name` must be lowercased.
-- The ``subpath`` is used to point to a subpath inside a package.
-- The ``version`` is often empty when a commit is not specified and should be
-  the commit in most cases when available.
+- The ``subpath`` is used to point to a package inside a module.
+- The ``version`` for modules should be semver with a v prefix, or a pseudo-version that points to an untagged commit.
 - Examples::
 
-      pkg:golang/github.com/gorilla/context@234fd47e07d1004f0aed9c
+      pkg:golang/github.com/gorilla/context@v1.1.1
       pkg:golang/google.golang.org/genproto#googleapis/api/annotations
-      pkg:golang/github.com/gorilla/context@234fd47e07d1004f0aed9c#api
```

Comment:
Please keep one of the github sha-1 hash examples in

Reply:
At least for modern Go tools, it should be a pseudo-version, not a bare commit ID like it is now: https://go.dev/doc/modules/version-numbers#in-development